.changesets/add-minutely-probes-system.md

```diff
@@ -0,0 +1,20 @@
+---
+bump: "minor"
+type: "add"
+---
+
+Add a minutely probes system. This can be used, alongside our metric helpers, to report metrics to AppSignal once per minute.
+
+```python
+from appsignal import probes, set_gauge
+
+def new_carts(previous_carts=None):
+    current_carts = Cart.objects.all().count()
+
+    if previous_carts is not None:
+      set_gauge("new_carts", current_carts - previous_carts)
+
+    return current_carts
+
+probes.register("new_carts", new_carts)
```

Comment:
@noccor I borrowed from your draft for this example :)

The probe taking an argument and returning a value is optional, the following is also okay:

```python
def new_carts():
  set_gauge("new_carts", ...)

probes.register("new_carts", new_carts)
```

Reply:
@unflxw - amazing! Thank you so much for getting this into action so quickly 🙇